src/m_pd.h

```diff
@@ -111,6 +111,11 @@ typedef unsigned __int64  uint64_t;
 # error invalid FLOATSIZE: must be 32 or 64
 #endif
 
+/* override for parallel processing support */
+#ifndef PD_PARALLEL
```

Comment:
Personally, I would call this PDPARALLEL without the underscore to better match the existing PDINSTANCE and PDTHREADS defines.

Reply:
Fair enough. On the other hand, there are quite a few constants/defines with a `PD_` prefix:
`PD_INTERNAL`, `PD_LONGINTTYPE`, `PD_FLOATSIZE`, `PD_BIGORSMALL`, etc. Personally, I think `PD_<name>` is better style, but I don't care too much :-)

Reply:
Generally, any suggestions for better naming are very much appreciated! (Naming things is hard...)